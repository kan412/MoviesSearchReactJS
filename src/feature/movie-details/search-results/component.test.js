import React from 'react';
import { shallow } from 'enzyme';
import SearchResults from './component';

describe('SearchResults', () => {
  it('renders properly', () => {
    const movies = [{}, {}];
    const component = shallow(<SearchResults data={movies} />);
    expect(component).toMatchSnapshot();
  });

  it('no films found', () => {
    const data = [];
    const component = shallow(<SearchResults data={data} />);
    expect(component.find('h3').text()).toBe('No films Found');
  });
});
