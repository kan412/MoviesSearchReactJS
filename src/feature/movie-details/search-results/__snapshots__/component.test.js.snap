// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResults renders properly 1`] = `
<div
  className="search-results-container"
>
  <Component />
  <Component />
</div>
`;
